2022-03-13 20:25:10,343 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:25:10,344 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['E:/GB_Code/Client-server_app/python_chat/server.py']
2022-03-13 20:25:10,346 DEBUG server.py Аргументы успешно загружены.
2022-03-13 20:25:10,347 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:25:11,126 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 20:25:44,827 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 20:25:55,929 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:25:55,929 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['E:/GB_Code/Client-server_app/python_chat/server.py']
2022-03-13 20:25:55,931 DEBUG server.py Аргументы успешно загружены.
2022-03-13 20:25:55,932 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:25:56,654 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 20:26:13,405 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 20:26:24,150 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:26:24,150 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['E:/GB_Code/Client-server_app/python_chat/server.py']
2022-03-13 20:26:24,153 DEBUG server.py Аргументы успешно загружены.
2022-03-13 20:26:24,154 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:26:24,294 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 20:30:16,904 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60283)
2022-03-13 20:30:17,411 DEBUG core.py Getting data from client exception.
Traceback (most recent call last):
  File "E:\GB_Code\Client-server_app\python_chat\server\core.py", line 87, in run
    get_message(client_with_message), client_with_message)
  File "E:\GB_Code\Client-server_app\python_chat\common\decorator.py", line 16, in log_saver
    ret = func_to_log(*args, **kwargs)
  File "E:\GB_Code\Client-server_app\python_chat\common\utils.py", line 19, in get_message
    response = json.loads(json_response)
  File "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2022-03-13 20:30:17,418 INFO core.py Клиент ('127.0.0.1', 60283) отключился от сервера.
2022-03-13 20:32:38,137 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60290)
2022-03-13 20:32:38,656 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1364, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60290)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:34:12,027 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 20:34:49,013 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:34:49,013 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['E:/GB_Code/Client-server_app/python_chat/server.py']
2022-03-13 20:34:49,015 DEBUG server.py Аргументы успешно загружены.
2022-03-13 20:34:49,016 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:34:49,736 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 20:34:50,121 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60301)
2022-03-13 20:34:50,636 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60301)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:35:55,076 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 20:36:00,304 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:36:00,304 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['E:/GB_Code/Client-server_app/python_chat/server.py']
2022-03-13 20:36:00,306 DEBUG server.py Аргументы успешно загружены.
2022-03-13 20:36:00,307 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:36:00,447 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 20:36:22,247 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60306)
2022-03-13 20:36:22,753 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60306)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:36:22,753 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647185782.2574046, 'user': {'account_name': 'Necris45', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAujzGyCZ1EXOLDOQ73nG2\ndmdzsCe2Y3G+MPDmGXiZe0aJ/JYpFPL3+lFVmkR+Mu9Ky0tQla9DmHkz/3tpGa7L\ng89H5RJCUtMHacIUcEHD4H2G3am+NBMo2UXVHAVykOxf44Spo1AAORsrmHE9EXJI\nZbh62HJbeFPFGyzaa5boeQRO0omeaPzx6oAgvI3lWe5SUtLaVwnoyE3wp9cM7u3g\ntPGocmC8lk84+s8OW7f2KN8WfT8rRwN7DB/rsgpAd5R7IXyCaYM1n58WSMbblFyB\nNsaK6m6tJK4McobK5t2La1LyI7oaqB95M7/ZWLknBxsmolkl2WACei1XKgVOINtP\nsQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 20:36:22,753 DEBUG core.py Start auth process for {'account_name': 'Necris45', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAujzGyCZ1EXOLDOQ73nG2\ndmdzsCe2Y3G+MPDmGXiZe0aJ/JYpFPL3+lFVmkR+Mu9Ky0tQla9DmHkz/3tpGa7L\ng89H5RJCUtMHacIUcEHD4H2G3am+NBMo2UXVHAVykOxf44Spo1AAORsrmHE9EXJI\nZbh62HJbeFPFGyzaa5boeQRO0omeaPzx6oAgvI3lWe5SUtLaVwnoyE3wp9cM7u3g\ntPGocmC8lk84+s8OW7f2KN8WfT8rRwN7DB/rsgpAd5R7IXyCaYM1n58WSMbblFyB\nNsaK6m6tJK4McobK5t2La1LyI7oaqB95M7/ZWLknBxsmolkl2WACei1XKgVOINtP\nsQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 20:36:22,757 DEBUG core.py Unknown username, sending {'response': 400, 'error': 'Пользователь не зарегистрирован.'}
2022-03-13 20:36:22,759 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60306)>, {'response': 400, 'error': 'Пользователь не зарегистрирован.'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:44:33,950 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60334)
2022-03-13 20:44:34,463 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60334)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:44:34,463 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647186273.960829, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 20:44:34,463 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 20:44:34,464 DEBUG core.py Correct username, starting passwd check.
2022-03-13 20:44:34,479 DEBUG core.py Auth message = {'response': 511, 'bin': 'b3535286f617213ec3ef73ceb3727c8f08066bc80723af4e14eadec64c2bb0aed700ed188f2c6ca91b9a151967ad5aaf8519344cc74e37cba5580b7ae694ad3d'}
2022-03-13 20:44:34,482 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60334)>, {'response': 511, 'bin': 'b3535286f617213ec3ef73ceb3727c8f08066bc80723af4e14eadec64c2bb0aed700ed188f2c6ca91b9a151967ad5aaf8519344cc74e37cba5580b7ae694ad3d'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:44:34,528 DEBUG core.py Getting data from client exception.
Traceback (most recent call last):
  File "E:\GB_Code\Client-server_app\python_chat\server\core.py", line 86, in run
    self.process_client_message(
  File "E:\GB_Code\Client-server_app\python_chat\common\decorator.py", line 60, in checker
    return func(*args, **kwargs)
  File "E:\GB_Code\Client-server_app\python_chat\server\core.py", line 145, in process_client_message
    self.autorize_user(message, client)
  File "E:\GB_Code\Client-server_app\python_chat\server\core.py", line 282, in autorize_user
    ans = get_message(sock)
  File "E:\GB_Code\Client-server_app\python_chat\common\decorator.py", line 17, in log_saver
    ret = func_to_log(*args, **kwargs)
  File "E:\GB_Code\Client-server_app\python_chat\common\utils.py", line 19, in get_message
    response = json.loads(json_response)
  File "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2022-03-13 20:44:34,530 INFO core.py Клиент ('127.0.0.1', 60334) отключился от сервера.
2022-03-13 20:45:19,009 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60347)
2022-03-13 20:45:19,515 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60347)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:45:19,515 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647186319.0208354, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 20:45:19,515 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 20:45:19,516 DEBUG core.py Correct username, starting passwd check.
2022-03-13 20:45:19,517 DEBUG core.py Auth message = {'response': 511, 'bin': 'eaf4cf3d0124922f6d5b88db8e8fbe3baa1c06099ed6ad0f044f726e4c00d0738512e4c26bafdefddee051e45685d135ad8fa66f4cfa04f26121a1e93053a323'}
2022-03-13 20:45:19,518 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60347)>, {'response': 511, 'bin': 'eaf4cf3d0124922f6d5b88db8e8fbe3baa1c06099ed6ad0f044f726e4c00d0738512e4c26bafdefddee051e45685d135ad8fa66f4cfa04f26121a1e93053a323'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:45:19,520 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60347)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:45:19,521 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60347)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:45:20,152 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60347)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:45:20,152 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647186319.521665, 'account_name': 'test1'}
2022-03-13 20:45:20,155 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60347)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 20:45:20,665 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60347)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:45:20,665 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647186320.2713885, 'user': 'test1'}
2022-03-13 20:45:20,682 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60347)>, {'response': 202, 'data_list': []}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 20:45:44,092 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60347)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:45:44,092 DEBUG core.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647186343.97243, 'account_name': 'test1'}
2022-03-13 20:45:44,092 INFO core.py Клиент ('127.0.0.1', 60347) отключился от сервера.
2022-03-13 20:46:08,120 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60353)
2022-03-13 20:46:08,624 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60353)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:46:08,624 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647186368.1300042, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 20:46:08,624 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 20:46:08,625 DEBUG core.py Correct username, starting passwd check.
2022-03-13 20:46:08,626 DEBUG core.py Auth message = {'response': 511, 'bin': 'b43b3261bf4eefa80a775cb6c017876389e9560f4ecb05c6e8cce27fe2fdda6679fcf5c5dc567e4706079c6af117e10acbf1528cff1fcf5a106cde2ff9aa2bb9'}
2022-03-13 20:46:08,628 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60353)>, {'response': 511, 'bin': 'b43b3261bf4eefa80a775cb6c017876389e9560f4ecb05c6e8cce27fe2fdda6679fcf5c5dc567e4706079c6af117e10acbf1528cff1fcf5a106cde2ff9aa2bb9'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:46:08,630 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60353)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:46:08,632 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60353)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:46:09,248 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60353)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:46:09,248 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647186368.6317735, 'account_name': 'test2'}
2022-03-13 20:46:09,250 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60353)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 20:46:09,763 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60353)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:46:09,763 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647186369.3736992, 'user': 'test2'}
2022-03-13 20:46:09,766 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=736, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60353)>, {'response': 202, 'data_list': []}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 20:46:13,233 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 20:46:45,572 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:46:45,572 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 20:46:45,575 DEBUG server.py Аргументы успешно загружены.
2022-03-13 20:46:45,576 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:46:45,717 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 20:48:39,257 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:48:39,257 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 20:48:39,260 DEBUG server.py Аргументы успешно загружены.
2022-03-13 20:48:39,260 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:48:39,413 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 20:49:01,551 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60381)
2022-03-13 20:49:02,065 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60381)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:49:02,065 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647186541.562698, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 20:49:02,065 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 20:49:02,069 DEBUG core.py Correct username, starting passwd check.
2022-03-13 20:49:02,070 DEBUG core.py Auth message = {'response': 511, 'bin': '7c9e7cb6359ba51f09f30150283617fdba539c2fb4a262aae1b9f5c959d42618080c11594807d81971123e629849d5789b1377c5a80032d6b4e21044ee6e6f9c'}
2022-03-13 20:49:02,072 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60381)>, {'response': 511, 'bin': '7c9e7cb6359ba51f09f30150283617fdba539c2fb4a262aae1b9f5c959d42618080c11594807d81971123e629849d5789b1377c5a80032d6b4e21044ee6e6f9c'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:49:02,073 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60381)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:49:02,075 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60381)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:49:02,749 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60381)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:49:02,749 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647186542.0749886, 'account_name': 'test1'}
2022-03-13 20:49:02,752 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60381)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 20:49:03,265 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60381)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:49:03,265 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647186542.8658688, 'user': 'test1'}
2022-03-13 20:49:03,269 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60381)>, {'response': 202, 'data_list': []}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 20:49:13,802 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60382)
2022-03-13 20:49:14,314 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60382)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:49:14,314 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647186553.8114142, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 20:49:14,314 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 20:49:14,315 DEBUG core.py Correct username, starting passwd check.
2022-03-13 20:49:14,316 DEBUG core.py Auth message = {'response': 511, 'bin': 'b130ca893f5b448ec007039529e7ded8a9f310ce443eaefbc9b7c3b0c18f61cd94d462843ddd01da689f235a009c82d186ac54365dee04444053bf6d0d023100'}
2022-03-13 20:49:14,317 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60382)>, {'response': 511, 'bin': 'b130ca893f5b448ec007039529e7ded8a9f310ce443eaefbc9b7c3b0c18f61cd94d462843ddd01da689f235a009c82d186ac54365dee04444053bf6d0d023100'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:49:14,319 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60382)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:49:14,321 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1336, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60382)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:56:21,532 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:56:21,533 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 20:56:21,535 DEBUG server.py Аргументы успешно загружены.
2022-03-13 20:56:21,536 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:56:21,686 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 20:56:39,288 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60419)
2022-03-13 20:56:39,802 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:56:39,802 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647186999.2972791, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 20:56:39,802 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 20:56:39,806 DEBUG core.py Correct username, starting passwd check.
2022-03-13 20:56:39,807 DEBUG core.py Auth message = {'response': 511, 'bin': '68913ab313abdd4bc7ccfa3acf4184df493aa52dea7b79fd6db57ef899eead0034d870167f7e8c553ce21d6af09267f570aece28530cfc769cdc67b3e21bfacb'}
2022-03-13 20:56:39,809 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>, {'response': 511, 'bin': '68913ab313abdd4bc7ccfa3acf4184df493aa52dea7b79fd6db57ef899eead0034d870167f7e8c553ce21d6af09267f570aece28530cfc769cdc67b3e21bfacb'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:56:39,811 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:56:39,813 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 20:56:40,472 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:56:40,472 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647186999.8122094, 'account_name': 'test1'}
2022-03-13 20:56:40,475 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 20:56:40,985 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:56:40,985 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647187000.6752145, 'user': 'test1'}
2022-03-13 20:56:40,990 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>, {'response': 202, 'data_list': []}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 20:56:45,580 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 20:56:45,580 DEBUG core.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647187005.2714555, 'user': 'test1', 'account_name': 'test2'}
2022-03-13 20:56:45,765 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60419)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 20:57:42,598 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:57:42,598 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['E:/GB_Code/Client-server_app/python_chat/server.py']
2022-03-13 20:57:42,600 DEBUG server.py Аргументы успешно загружены.
2022-03-13 20:57:42,601 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 20:57:42,753 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 20:58:50,412 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 21:00:47,451 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:00:47,451 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:00:47,453 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:00:47,453 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:00:47,603 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:01:05,794 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60450)
2022-03-13 21:01:06,319 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:01:06,319 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647187265.8194864, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:01:06,319 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:01:06,324 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:01:06,325 DEBUG core.py Auth message = {'response': 511, 'bin': 'da0855f4027d93eb31c02b61f2ac701f6c07aff40398f2a8c365d4c21f6a4c7ce5a074a0a57d5e6fbbbc0087d76d31e0ddb1db5c9c61b5f6d50e152b1191fde3'}
2022-03-13 21:01:06,327 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>, {'response': 511, 'bin': 'da0855f4027d93eb31c02b61f2ac701f6c07aff40398f2a8c365d4c21f6a4c7ce5a074a0a57d5e6fbbbc0087d76d31e0ddb1db5c9c61b5f6d50e152b1191fde3'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:01:06,328 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:01:06,330 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:01:06,969 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:01:06,969 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647187266.3309643, 'account_name': 'test1'}
2022-03-13 21:01:06,972 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:01:07,502 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:01:07,502 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647187267.1095266, 'user': 'test1'}
2022-03-13 21:01:07,507 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>, {'response': 202, 'data_list': ['test2']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:01:23,833 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60452)
2022-03-13 21:01:24,338 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:01:24,338 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647187283.844381, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:01:24,338 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:01:24,339 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:01:24,340 DEBUG core.py Auth message = {'response': 511, 'bin': '5ce30affa99b598da5964a93c8cd78133149854da61f6006c40465e55e78162838c36841956b5369fbe226a92cdbd22146adc941817c01017c7c9174c7f5159b'}
2022-03-13 21:01:24,341 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>, {'response': 511, 'bin': '5ce30affa99b598da5964a93c8cd78133149854da61f6006c40465e55e78162838c36841956b5369fbe226a92cdbd22146adc941817c01017c7c9174c7f5159b'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:01:24,343 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:01:24,345 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:01:25,004 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:01:25,004 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647187284.3445096, 'account_name': 'test2'}
2022-03-13 21:01:25,006 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:01:25,518 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:01:25,518 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647187285.1377692, 'user': 'test2'}
2022-03-13 21:01:25,521 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>, {'response': 202, 'data_list': []}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:01:36,700 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:01:36,700 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647187296.6619668, 'account_name': 'test2'}
2022-03-13 21:01:36,703 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:01:47,419 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:01:47,419 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1647187306.8813376, 'mess_text': 'VHv5aaBeMavogGHwhIt0EbJiEHseBb14o95GQyT5fB3UIOF5Zr5GwnNWQW7gAVYS83vPkRv0sqNRcUQuraUd/Bs30ttkQcCGUzP6p7frqooIDwgG3+u+V05S/EL/lLFWKUGY9WVGhjD8qbEqsrOtq6VFWyW2haXHdlJTP951AFABt0poXgKmeNOjpinOtV2d47mDadWiXAbc41vjL/6q2STikR2bx0GJeoHaSdOvyWrboG8fKVkC23NUGzPzI6zP6MkBuEi2rWPKZL9Jneu+Is1kObjb3NOuirAHYFqHmIg3OJT8/0BWEaGECTfJZknIUuBBjM0MZzVIgKHjYmfR/g=='}
2022-03-13 21:01:47,564 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>, {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1647187306.8813376, 'mess_text': 'VHv5aaBeMavogGHwhIt0EbJiEHseBb14o95GQyT5fB3UIOF5Zr5GwnNWQW7gAVYS83vPkRv0sqNRcUQuraUd/Bs30ttkQcCGUzP6p7frqooIDwgG3+u+V05S/EL/lLFWKUGY9WVGhjD8qbEqsrOtq6VFWyW2haXHdlJTP951AFABt0poXgKmeNOjpinOtV2d47mDadWiXAbc41vjL/6q2STikR2bx0GJeoHaSdOvyWrboG8fKVkC23NUGzPzI6zP6MkBuEi2rWPKZL9Jneu+Is1kObjb3NOuirAHYFqHmIg3OJT8/0BWEaGECTfJZknIUuBBjM0MZzVIgKHjYmfR/g=='}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_message.
2022-03-13 21:01:47,564 INFO core.py Отправлено сообщение пользователю test2 от пользователя test1.
2022-03-13 21:01:47,566 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:06:12,920 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1392, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60450)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:06:12,920 DEBUG core.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647187572.795265, 'account_name': 'test1'}
2022-03-13 21:06:12,920 INFO core.py Клиент ('127.0.0.1', 60450) отключился от сервера.
2022-03-13 21:06:14,083 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60452)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:06:14,083 DEBUG core.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647187573.7394304, 'account_name': 'test2'}
2022-03-13 21:06:14,083 INFO core.py Клиент ('127.0.0.1', 60452) отключился от сервера.
2022-03-13 21:06:46,725 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60481)
2022-03-13 21:06:47,230 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1416, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60481)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:06:47,230 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647187606.736794, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:06:47,230 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:06:47,231 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:06:47,232 DEBUG core.py Auth message = {'response': 511, 'bin': 'cc1a499a0562f564fa1fcf8d151f10c8ce5e69bc87d4abe706ac6b647e53737781fcd033968b0804cac9fb9605bb2d41e49f43751f58e4265e68f7b5e494055f'}
2022-03-13 21:06:47,234 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1416, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60481)>, {'response': 511, 'bin': 'cc1a499a0562f564fa1fcf8d151f10c8ce5e69bc87d4abe706ac6b647e53737781fcd033968b0804cac9fb9605bb2d41e49f43751f58e4265e68f7b5e494055f'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:06:47,235 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1416, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60481)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:06:47,237 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1416, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60481)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:06:47,902 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1416, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60481)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:06:47,902 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647187607.237456, 'account_name': 'test2'}
2022-03-13 21:06:47,904 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1416, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60481)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:06:48,419 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1416, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60481)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:06:48,419 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647187608.0529773, 'user': 'test2'}
2022-03-13 21:06:48,422 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1416, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60481)>, {'response': 202, 'data_list': []}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:07:34,229 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60485)
2022-03-13 21:07:34,733 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:07:34,733 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647187654.2404394, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:07:34,733 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:07:34,734 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:07:34,735 DEBUG core.py Auth message = {'response': 511, 'bin': 'df26cf688a41f13876d96b7ec1e634e063c77857e01aaab475363a737a6483b7ad7031fa8991116cb9c03b7daa08a579468982ae803d53088abe36f46f13afce'}
2022-03-13 21:07:34,737 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>, {'response': 511, 'bin': 'df26cf688a41f13876d96b7ec1e634e063c77857e01aaab475363a737a6483b7ad7031fa8991116cb9c03b7daa08a579468982ae803d53088abe36f46f13afce'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:07:34,738 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:07:34,740 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:07:35,370 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:07:35,370 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647187654.7406666, 'account_name': 'test1'}
2022-03-13 21:07:35,373 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:07:35,887 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:07:35,887 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647187655.5012875, 'user': 'test1'}
2022-03-13 21:07:35,890 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>, {'response': 202, 'data_list': ['test2']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:07:37,929 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:07:37,929 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647187657.6837316, 'account_name': 'test2'}
2022-03-13 21:07:37,932 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:07:45,554 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60485)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:07:45,554 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1647187665.2905073, 'mess_text': '1HTE39l6055tnu6HCohqg2q+z38whE2qy9hZMOlXp4Nu6xxX5HcYm13F3aBgY9B0M6R/x+/02iaezf4nkgKNj/x75ktvAS6LUBJZ83cPflwx21XznYOKbhfLkJ1noGKLhp9aLIy5aaSEPSKKHW1dpLtLRVeS/oXDf9nmG6hCkdCWywcLxDh0Hq/btOBtsuOgroDzNDOiPbXm4gk2EAA/TaoakkVya+PUVWxn+ZPsMZF05SIjR3GJTRrD/jcYKnInKSHy3U9/UNPZu5GVGXZXdQNexQCBqV//qa2RHOjqMxO21hwf5DLFMibLrNIGTyuN06f0VzJ6puVo7hj9BXmmkA=='}
2022-03-13 21:09:02,260 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:09:02,261 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:09:02,262 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:09:02,263 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:09:02,483 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:09:18,316 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60494)
2022-03-13 21:09:18,827 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:09:18,827 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647187758.3275604, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:09:18,827 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:09:18,831 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:09:18,833 DEBUG core.py Auth message = {'response': 511, 'bin': 'b14135b1c3488a22b421c3855cdb2e43e74ee45dd4711f0d3cdfcef5dcf02d8d4b022c54b3aff487f8c269b3b4fd0298bd4c76e40219aa20881651d863df5396'}
2022-03-13 21:09:18,835 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>, {'response': 511, 'bin': 'b14135b1c3488a22b421c3855cdb2e43e74ee45dd4711f0d3cdfcef5dcf02d8d4b022c54b3aff487f8c269b3b4fd0298bd4c76e40219aa20881651d863df5396'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:09:18,836 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:09:18,838 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:09:19,543 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:09:19,543 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647187758.8384829, 'account_name': 'test1'}
2022-03-13 21:09:19,546 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:09:20,060 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:09:20,060 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647187759.6737669, 'user': 'test1'}
2022-03-13 21:09:20,065 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>, {'response': 202, 'data_list': ['test2']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:09:23,639 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:09:23,639 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647187762.9613154, 'account_name': 'test2'}
2022-03-13 21:09:23,642 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:09:24,153 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:09:24,153 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647187763.6555157, 'account_name': 'test2'}
2022-03-13 21:09:24,156 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1372, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60494)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:10:03,416 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60495)
2022-03-13 21:10:03,929 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:10:03,929 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647187803.427425, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:10:03,929 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:10:03,930 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:10:03,931 DEBUG core.py Auth message = {'response': 511, 'bin': '611cda92fb09e6a7b6f83b03bebfa0327eab9bce5233eb24a5dcbfa07c9ee0614e71f90c5eee9b52aea26e2e61422e2185817255ebd2dbbdbb734a8e95117f51'}
2022-03-13 21:10:03,933 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>, {'response': 511, 'bin': '611cda92fb09e6a7b6f83b03bebfa0327eab9bce5233eb24a5dcbfa07c9ee0614e71f90c5eee9b52aea26e2e61422e2185817255ebd2dbbdbb734a8e95117f51'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:10:03,934 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:10:03,936 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:10:04,566 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:10:04,566 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647187803.9358478, 'account_name': 'test2'}
2022-03-13 21:10:04,568 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:10:05,081 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:10:05,081 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647187804.7335122, 'user': 'test2'}
2022-03-13 21:10:05,084 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>, {'response': 202, 'data_list': []}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:10:09,181 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:10:09,181 DEBUG core.py Разбор сообщения от клиента : {'action': 'add', 'time': 1647187808.9905958, 'user': 'test2', 'account_name': 'test1'}
2022-03-13 21:10:09,275 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:10:11,826 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:10:11,826 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647187811.593587, 'account_name': 'test1'}
2022-03-13 21:10:11,829 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:10:19,486 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1380, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60495)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:10:19,486 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1647187819.1931736, 'mess_text': 'OtQH/juLnjte2T8jV1k9HiwUCVfD/aXjAQ73+n2NLYbEmtU4geC6pxO4z4UCz9OHRlU8XLjBCsKbrmcNJ7t96TiNT+khuWyAiBqrAGl8PjInUuogepqhn8EkWhd6nxnzF3mAHX75m09YEnM5ceza08b+lZge5tVK5Bfj7n7jdqCpr7zZwNOLguVtyhXBHcQhvCdO0JUhim0RVfsNNNapI9s0oDo4zy4AGJFZ0Kwq4oZRs5stNxX9atmrEoVnNsqudcPQXHsb8QsejTKaLCneV4NjcXRaqi9YdTmdZWZdpLpBlVPSPJr0wLGZKuF7TMCfhP29SX0ONe+TbwrJSK8OiQ=='}
2022-03-13 21:12:47,273 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:12:47,273 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:12:47,274 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:12:47,275 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:12:47,425 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:13:14,974 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60502)
2022-03-13 21:13:15,477 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60502)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:13:15,477 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647187994.9847982, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:13:15,477 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:13:15,482 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:13:15,483 DEBUG core.py Auth message = {'response': 511, 'bin': 'c52daa3b724c9dcdc7480306aff094a639973d88f4411d014ffc30b887cc2bea875b205b936f5efb90406c27392a4e7a2954daf25d95b3244e0a92327945e3fe'}
2022-03-13 21:13:15,485 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60502)>, {'response': 511, 'bin': 'c52daa3b724c9dcdc7480306aff094a639973d88f4411d014ffc30b887cc2bea875b205b936f5efb90406c27392a4e7a2954daf25d95b3244e0a92327945e3fe'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:13:15,486 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60502)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:13:15,488 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60502)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:13:16,146 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60502)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:13:16,146 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647187995.4884045, 'account_name': 'test1'}
2022-03-13 21:13:16,149 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60502)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:13:16,661 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60502)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:13:16,661 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647187996.288626, 'user': 'test1'}
2022-03-13 21:13:16,665 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60502)>, {'response': 202, 'data_list': ['test2']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:13:29,107 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60505)
2022-03-13 21:13:29,623 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:13:29,623 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647188009.1183035, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:13:29,623 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:13:29,624 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:13:29,625 DEBUG core.py Auth message = {'response': 511, 'bin': 'db7f539a4b5adec3c460302bd2c2756a2161cb11aade73b906035925f15f4ce744929f8e4fcd15701623a6af925c1a08a0e5bbecf3481f5d89d819d613a59ee1'}
2022-03-13 21:13:29,627 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>, {'response': 511, 'bin': 'db7f539a4b5adec3c460302bd2c2756a2161cb11aade73b906035925f15f4ce744929f8e4fcd15701623a6af925c1a08a0e5bbecf3481f5d89d819d613a59ee1'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:13:29,628 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:13:29,630 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:13:30,295 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:13:30,295 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647188009.6300733, 'account_name': 'test2'}
2022-03-13 21:13:30,297 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:13:30,807 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:13:30,807 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647188010.4659042, 'user': 'test2'}
2022-03-13 21:13:30,810 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>, {'response': 202, 'data_list': ['test1']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:13:32,863 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:13:32,863 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647188012.346435, 'account_name': 'test1'}
2022-03-13 21:13:32,866 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:13:40,480 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:13:40,480 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1647188020.338486, 'mess_text': 'cYj9DhwJjPhiCcZX1cc5nRrubP+xu7ob8vtBtxWnslibK6TQWtW5aI6m3iSmb0GmST2+4OUnnT84TkAGNOVuxxiRRudruah7KcJhC3fWM8RLRu0MH18kDvayT76FokJdf5bJpcYeGCsouHCVF6TaThuNxQirYz5zC/q6d/ZPnYWXDvJ7McIrSaQjS93DupHEhOB9n0UGoWGJcOEVf74HTgKrvCM1fw9C4sqeGRUoZGPs1qBr7M7rhb576UO4Jr4sTR68KKqonpXc1lZ4gv/RPNIBZtjdvI0xdT/JB2DAenkH/xynO0KXP7gdeE7VKObkLTPwIh1CPaWF3RUbIiXx2A=='}
2022-03-13 21:13:40,607 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1344, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60502)>, {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1647188020.338486, 'mess_text': 'cYj9DhwJjPhiCcZX1cc5nRrubP+xu7ob8vtBtxWnslibK6TQWtW5aI6m3iSmb0GmST2+4OUnnT84TkAGNOVuxxiRRudruah7KcJhC3fWM8RLRu0MH18kDvayT76FokJdf5bJpcYeGCsouHCVF6TaThuNxQirYz5zC/q6d/ZPnYWXDvJ7McIrSaQjS93DupHEhOB9n0UGoWGJcOEVf74HTgKrvCM1fw9C4sqeGRUoZGPs1qBr7M7rhb576UO4Jr4sTR68KKqonpXc1lZ4gv/RPNIBZtjdvI0xdT/JB2DAenkH/xynO0KXP7gdeE7VKObkLTPwIh1CPaWF3RUbIiXx2A=='}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_message.
2022-03-13 21:13:40,607 INFO core.py Отправлено сообщение пользователю test1 от пользователя test2.
2022-03-13 21:13:40,609 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1352, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60505)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:13:41,629 DEBUG core.py Getting data from client exception.
Traceback (most recent call last):
  File "E:\GB_Code\Client-server_app\python_chat\server\core.py", line 87, in run
    get_message(client_with_message), client_with_message)
  File "E:\GB_Code\Client-server_app\python_chat\common\decorator.py", line 17, in log_saver
    ret = func_to_log(*args, **kwargs)
  File "E:\GB_Code\Client-server_app\python_chat\common\utils.py", line 17, in get_message
    encoded_response = client.recv(MAX_PACKAGE_LENGTH)
ConnectionResetError: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
2022-03-13 21:13:41,630 INFO core.py Клиент ('127.0.0.1', 60502) отключился от сервера.
2022-03-13 21:19:23,467 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:19:23,467 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:19:23,468 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:19:23,469 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:19:23,639 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:19:39,819 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60572)
2022-03-13 21:19:40,333 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60572)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:19:40,333 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647188379.8297408, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:19:40,333 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:19:40,338 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:19:40,339 DEBUG core.py Auth message = {'response': 511, 'bin': 'fef5e24e3208233541c8e357f92fe6babfdd574c8630ae1634eb46a815eea71b6eba5a5eef1a7a251f1237c9ce5f9f3a740a872200d38c9e0bc5d2063a5b200d'}
2022-03-13 21:19:40,341 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60572)>, {'response': 511, 'bin': 'fef5e24e3208233541c8e357f92fe6babfdd574c8630ae1634eb46a815eea71b6eba5a5eef1a7a251f1237c9ce5f9f3a740a872200d38c9e0bc5d2063a5b200d'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:19:40,342 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60572)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:19:40,344 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60572)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:19:41,017 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60572)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:19:41,017 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647188380.3440268, 'account_name': 'test1'}
2022-03-13 21:19:41,020 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60572)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:19:41,528 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60572)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:19:41,528 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647188381.1342015, 'user': 'test1'}
2022-03-13 21:19:41,533 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60572)>, {'response': 202, 'data_list': ['test2']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:19:42,034 DEBUG core.py Getting data from client exception.
Traceback (most recent call last):
  File "E:\GB_Code\Client-server_app\python_chat\server\core.py", line 87, in run
    get_message(client_with_message), client_with_message)
  File "E:\GB_Code\Client-server_app\python_chat\common\decorator.py", line 17, in log_saver
    ret = func_to_log(*args, **kwargs)
  File "E:\GB_Code\Client-server_app\python_chat\common\utils.py", line 17, in get_message
    encoded_response = client.recv(MAX_PACKAGE_LENGTH)
ConnectionResetError: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
2022-03-13 21:19:42,035 INFO core.py Клиент ('127.0.0.1', 60572) отключился от сервера.
2022-03-13 21:21:19,318 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60578)
2022-03-13 21:21:19,833 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1376, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60578)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:21:19,833 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647188479.3282497, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:21:19,833 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:21:19,834 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:21:19,835 DEBUG core.py Auth message = {'response': 511, 'bin': '7e51820b01549a0de9f23931c2190b7803869decf95dc8a5af24267d364a2336e2aeed5430552e3021447ebd6d7a8ca6befaeb50cd7197b8cf1969ed7d3c2b13'}
2022-03-13 21:21:19,837 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1376, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60578)>, {'response': 511, 'bin': '7e51820b01549a0de9f23931c2190b7803869decf95dc8a5af24267d364a2336e2aeed5430552e3021447ebd6d7a8ca6befaeb50cd7197b8cf1969ed7d3c2b13'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:21:19,838 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1376, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60578)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:21:19,840 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1376, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60578)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:21:20,543 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1376, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60578)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:21:20,543 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647188479.8406343, 'account_name': 'test1'}
2022-03-13 21:21:20,545 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1376, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60578)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:21:21,062 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1376, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60578)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:21:21,062 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647188480.673488, 'user': 'test1'}
2022-03-13 21:21:21,065 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1376, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60578)>, {'response': 202, 'data_list': ['test2']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:21:21,575 DEBUG core.py Getting data from client exception.
Traceback (most recent call last):
  File "E:\GB_Code\Client-server_app\python_chat\server\core.py", line 87, in run
    get_message(client_with_message), client_with_message)
  File "E:\GB_Code\Client-server_app\python_chat\common\decorator.py", line 17, in log_saver
    ret = func_to_log(*args, **kwargs)
  File "E:\GB_Code\Client-server_app\python_chat\common\utils.py", line 17, in get_message
    encoded_response = client.recv(MAX_PACKAGE_LENGTH)
ConnectionResetError: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
2022-03-13 21:21:21,575 INFO core.py Клиент ('127.0.0.1', 60578) отключился от сервера.
2022-03-13 21:26:27,579 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:26:27,579 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:26:27,580 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:26:27,581 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:26:27,797 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:26:39,433 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 21:27:44,904 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:27:44,904 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:27:44,905 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:27:44,906 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:27:45,076 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:28:21,696 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60610)
2022-03-13 21:28:22,203 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:28:22,203 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647188901.7156398, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:28:22,203 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:28:22,208 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:28:22,222 DEBUG core.py Auth message = {'response': 511, 'bin': '4deee51eeaa72c53eab56a38f4a609e5c6de884d9baedaf2266327fa92f36386cfb2ba306a26309a94b3ab4eb2825dc6b44269042afd582211c990d23d11ac04'}
2022-03-13 21:28:22,224 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>, {'response': 511, 'bin': '4deee51eeaa72c53eab56a38f4a609e5c6de884d9baedaf2266327fa92f36386cfb2ba306a26309a94b3ab4eb2825dc6b44269042afd582211c990d23d11ac04'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:28:22,225 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:28:22,227 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:28:22,876 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:28:22,876 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647188902.2263012, 'account_name': 'test1'}
2022-03-13 21:28:22,879 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:28:23,394 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:28:23,394 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647188903.0757437, 'user': 'test1'}
2022-03-13 21:28:23,399 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>, {'response': 202, 'data_list': ['test2']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:29:38,301 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:29:38,301 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647188978.2390094, 'account_name': 'test2'}
2022-03-13 21:29:38,303 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:30:07,856 INFO core.py Установлено соедение с ПК ('127.0.0.1', 60619)
2022-03-13 21:30:08,369 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:30:08,369 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647189007.8668575, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:30:08,369 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:30:08,370 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:30:08,371 DEBUG core.py Auth message = {'response': 511, 'bin': '606aa2fc58985b440d4b3d92d45247d098cce4e7c875b7b5ecedfb790e8cc8f35ae3b63d84758240ca13db8f57bbc5a003e84f9bf2d238a4170b5243e9130c64'}
2022-03-13 21:30:08,373 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>, {'response': 511, 'bin': '606aa2fc58985b440d4b3d92d45247d098cce4e7c875b7b5ecedfb790e8cc8f35ae3b63d84758240ca13db8f57bbc5a003e84f9bf2d238a4170b5243e9130c64'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:30:08,374 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:30:08,376 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:30:09,027 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:30:09,027 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647189008.3767695, 'account_name': 'test2'}
2022-03-13 21:30:09,032 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:30:09,541 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:30:09,541 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647189009.162303, 'user': 'test2'}
2022-03-13 21:30:09,544 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>, {'response': 202, 'data_list': ['test1']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:30:12,106 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:30:12,106 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647189012.0337076, 'account_name': 'test1'}
2022-03-13 21:30:12,108 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:30:21,291 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:30:21,291 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1647189020.9350498, 'mess_text': 'mSHT6h2+a+7R91l3bRblznqz0Chnry7cmm2/NjCeFuBkHsEgmDvrrwczxum+aOSB5PZ5QAi9yaY6JxRFc4tbvusuz7VZhRCt39nBf6LzODznbaIDe7DAVRqnBfKE2JRIScwVn9gOlQ5D+DUtOQtLA4ichaBftSPVIjOSTzHjqtqP/R6H6DtdA04BHj3Ce9XCesf3IvQo/m9tKKbNYycm8bjbAly3cLQPSkN/pOlCXqwpxDP4xgwPQaWMnxjtIWLwSpDYovdvXS2c+KK+tL+F5j56yxgMx7Dke8F23b6HCWaSGPvwCOj13G2MxZCDAIWH0vtGPm/r46Gg4Q7t31PDVQ=='}
2022-03-13 21:30:21,409 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>, {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1647189020.9350498, 'mess_text': 'mSHT6h2+a+7R91l3bRblznqz0Chnry7cmm2/NjCeFuBkHsEgmDvrrwczxum+aOSB5PZ5QAi9yaY6JxRFc4tbvusuz7VZhRCt39nBf6LzODznbaIDe7DAVRqnBfKE2JRIScwVn9gOlQ5D+DUtOQtLA4ichaBftSPVIjOSTzHjqtqP/R6H6DtdA04BHj3Ce9XCesf3IvQo/m9tKKbNYycm8bjbAly3cLQPSkN/pOlCXqwpxDP4xgwPQaWMnxjtIWLwSpDYovdvXS2c+KK+tL+F5j56yxgMx7Dke8F23b6HCWaSGPvwCOj13G2MxZCDAIWH0vtGPm/r46Gg4Q7t31PDVQ=='}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_message.
2022-03-13 21:30:21,409 INFO core.py Отправлено сообщение пользователю test2 от пользователя test1.
2022-03-13 21:30:21,411 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:30:30,610 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:30:30,610 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1647189030.2927864, 'mess_text': 'guaZeH3Pxv+eqd25urqj9I0yydKjr66064r8pwYQT47zZuT82nxg2gB4Yvq2LpJB16zmvDhBNRNYv08Hs1IwT5bhV1kGkLZZ/YTqieV0hroysRjmWnkASnEz/i0259fn+31g+CanPI9OKQKq/WBONqAOMMCUA/U9I/dUqv73dbAkDcRWkcv5pJxHJtsy+9LF8UC460O10FYlfdh7LSULTc9gcnucPRL3FUrlFFyhS3xjH4+hW0UQ6tcVxUVDaNHaHre2GatGxAukk/kpNghUThSGUUUi9HJnEG1BtiO2r6nbHepa21ICEzkmjhln1ukoDLRUXGLJs7OIvZ5pAg34pw=='}
2022-03-13 21:30:30,735 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>, {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1647189030.2927864, 'mess_text': 'guaZeH3Pxv+eqd25urqj9I0yydKjr66064r8pwYQT47zZuT82nxg2gB4Yvq2LpJB16zmvDhBNRNYv08Hs1IwT5bhV1kGkLZZ/YTqieV0hroysRjmWnkASnEz/i0259fn+31g+CanPI9OKQKq/WBONqAOMMCUA/U9I/dUqv73dbAkDcRWkcv5pJxHJtsy+9LF8UC460O10FYlfdh7LSULTc9gcnucPRL3FUrlFFyhS3xjH4+hW0UQ6tcVxUVDaNHaHre2GatGxAukk/kpNghUThSGUUUi9HJnEG1BtiO2r6nbHepa21ICEzkmjhln1ukoDLRUXGLJs7OIvZ5pAg34pw=='}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_message.
2022-03-13 21:30:30,735 INFO core.py Отправлено сообщение пользователю test1 от пользователя test2.
2022-03-13 21:30:30,736 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:30:33,813 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1100, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60610)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:30:33,813 DEBUG core.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647189033.8071134, 'account_name': 'test1'}
2022-03-13 21:30:33,813 INFO core.py Клиент ('127.0.0.1', 60610) отключился от сервера.
2022-03-13 21:30:35,456 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1088, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 60619)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:30:35,456 DEBUG core.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647189034.8666818, 'account_name': 'test2'}
2022-03-13 21:30:35,456 INFO core.py Клиент ('127.0.0.1', 60619) отключился от сервера.
2022-03-13 21:31:15,190 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:31:15,190 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:31:15,192 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:31:15,193 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:31:15,335 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:31:27,447 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 21:31:28,126 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 21:31:32,660 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:31:32,660 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:31:32,663 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:31:32,664 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:31:32,807 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:33:01,755 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:33:01,755 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:33:01,758 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:33:01,759 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:33:01,905 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:33:11,805 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
2022-03-13 21:39:48,701 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:39:48,702 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:39:48,704 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:39:48,705 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:39:48,855 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:39:57,866 INFO core.py Установлено соедение с ПК ('127.0.0.1', 50690)
2022-03-13 21:39:57,960 INFO core.py Установлено соедение с ПК ('127.0.0.1', 50691)
2022-03-13 21:39:57,963 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50690)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:39:57,963 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647189597.8763697, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:39:57,963 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:39:57,967 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:39:57,968 DEBUG core.py Auth message = {'response': 511, 'bin': '3c0b95e22533c29b36b221c49d9215dd91bdd679e258d07fb02e833fbb9d95fb04efcd84397c6b40dda2bcdf155e68ac152a837a483568e624d924829a0bbae4'}
2022-03-13 21:39:57,970 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50690)>, {'response': 511, 'bin': '3c0b95e22533c29b36b221c49d9215dd91bdd679e258d07fb02e833fbb9d95fb04efcd84397c6b40dda2bcdf155e68ac152a837a483568e624d924829a0bbae4'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:39:57,972 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50690)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:39:57,973 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50690)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:39:58,613 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50690)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:39:58,613 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647189597.9738686, 'account_name': 'test2'}
2022-03-13 21:39:58,616 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50690)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:39:58,618 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1304, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50691)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:39:58,618 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647189597.9718678, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:39:58,618 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:39:58,619 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:39:58,619 DEBUG core.py Auth message = {'response': 511, 'bin': 'f2a0567c262b610f67c721eb9debc8ebfe4353ff82be192f2b72eb2881baef92a36f16dea6c66930ff028f9a72c5c9c4a3e88f325bdb7565fecfaf8b93a62167'}
2022-03-13 21:39:58,621 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1304, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50691)>, {'response': 511, 'bin': 'f2a0567c262b610f67c721eb9debc8ebfe4353ff82be192f2b72eb2881baef92a36f16dea6c66930ff028f9a72c5c9c4a3e88f325bdb7565fecfaf8b93a62167'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:39:58,623 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1304, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50691)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:39:58,624 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1304, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50691)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:44:27,695 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:44:27,695 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:44:27,696 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:44:27,697 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:44:27,846 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:45:06,339 INFO core.py Установлено соедение с ПК ('127.0.0.1', 50719)
2022-03-13 21:45:06,849 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50719)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:45:06,849 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647189906.3491538, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:45:06,849 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:45:06,855 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:45:06,876 DEBUG core.py Auth message = {'response': 511, 'bin': 'aa2f5943b4a9b647fdcaa394a7a3f75da8c5338a12e9ef8f0115a6c90dd18cdac25ec6b4b2f7c0b6a9a58af77d3e4cc6bf1c6b8cb6ed40a0799120fad16baf77'}
2022-03-13 21:45:06,881 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50719)>, {'response': 511, 'bin': 'aa2f5943b4a9b647fdcaa394a7a3f75da8c5338a12e9ef8f0115a6c90dd18cdac25ec6b4b2f7c0b6a9a58af77d3e4cc6bf1c6b8cb6ed40a0799120fad16baf77'}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:45:06,883 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50719)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:45:06,885 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50719)>, {'response': 200}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:45:07,541 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50719)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:45:07,541 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647189906.8851962, 'account_name': 'test1'}
2022-03-13 21:45:07,544 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50719)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:45:08,055 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50719)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:45:08,055 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647189907.6656768, 'user': 'test1'}
2022-03-13 21:45:08,060 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50719)>, {'response': 202, 'data_list': ['test2']}), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:45:14,666 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1032, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 50719)>,), {}.
Вызов из модуля "C:\Python310\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:45:14,666 DEBUG core.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647189914.4900377, 'account_name': 'test1'}
2022-03-13 21:45:14,666 INFO core.py Клиент ('127.0.0.1', 50719) отключился от сервера.
2022-03-13 21:46:25,531 DEBUG decorator.py Была вызвана функция config_load c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:46:25,531 DEBUG server.py Инициализация парсера аргументов коммандной строки: ['server.py']
2022-03-13 21:46:25,534 DEBUG server.py Аргументы успешно загружены.
2022-03-13 21:46:25,535 DEBUG decorator.py Была вызвана функция arg_parser c параметрами ('7777', ''), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции main.
2022-03-13 21:46:25,679 INFO core.py Запущен сервер, порт для подключений: 7777 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2022-03-13 21:46:35,511 INFO core.py Установлено соедение с ПК ('127.0.0.1', 64264)
2022-03-13 21:46:36,016 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:46:36,016 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647189995.5216045, 'user': {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:46:36,016 DEBUG core.py Start auth process for {'account_name': 'test1', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:46:36,020 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:46:36,020 DEBUG core.py Auth message = {'response': 511, 'bin': 'f65f456beb148df99c6b338ff478c41a53c86767b970b1c4ac0e8e1755602311443e9d137d0f23027409cd40956017a93dabbb5d06e493d2a8c79a03ad382077'}
2022-03-13 21:46:36,023 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>, {'response': 511, 'bin': 'f65f456beb148df99c6b338ff478c41a53c86767b970b1c4ac0e8e1755602311443e9d137d0f23027409cd40956017a93dabbb5d06e493d2a8c79a03ad382077'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:46:36,025 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:46:36,025 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:46:36,674 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:46:36,674 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647189996.0259893, 'account_name': 'test1'}
2022-03-13 21:46:36,677 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:46:37,188 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:46:37,188 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647189996.8170006, 'user': 'test1'}
2022-03-13 21:46:37,193 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>, {'response': 202, 'data_list': ['test2']}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:46:40,585 INFO core.py Установлено соедение с ПК ('127.0.0.1', 64265)
2022-03-13 21:46:41,092 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:46:41,092 DEBUG core.py Разбор сообщения от клиента : {'action': 'presence', 'time': 1647190000.5952926, 'user': {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}}
2022-03-13 21:46:41,092 DEBUG core.py Start auth process for {'account_name': 'test2', 'pubkey': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}
2022-03-13 21:46:41,093 DEBUG core.py Correct username, starting passwd check.
2022-03-13 21:46:41,094 DEBUG core.py Auth message = {'response': 511, 'bin': '04a8d1c1e6286f57e21033ab30f5a9778db085ae6b41f148e0ca8c6904ec82fb9276820f105451364b07ef01f569dd001f1950b64a9ee9924a6ed3d20bef7311'}
2022-03-13 21:46:41,096 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>, {'response': 511, 'bin': '04a8d1c1e6286f57e21033ab30f5a9778db085ae6b41f148e0ca8c6904ec82fb9276820f105451364b07ef01f569dd001f1950b64a9ee9924a6ed3d20bef7311'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:46:41,097 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:46:41,099 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции autorize_user.
2022-03-13 21:46:41,790 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:46:41,790 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_users', 'time': 1647190001.0982647, 'account_name': 'test2'}
2022-03-13 21:46:41,792 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>, {'response': 202, 'data_list': ['test1', 'test2', 'test3']}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:46:42,303 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:46:42,303 DEBUG core.py Разбор сообщения от клиента : {'action': 'get_contacts', 'time': 1647190001.954327, 'user': 'test2'}
2022-03-13 21:46:42,306 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>, {'response': 202, 'data_list': ['test1']}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:46:47,411 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:46:47,411 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647190007.3311527, 'account_name': 'test1'}
2022-03-13 21:46:47,413 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAmJYKvEhYYV0fe5Q8eQB1\nn+hdUc8LfAAQ9a+YzbvpBojsR7cHAmKf5iAXzdIiPpxBe0jHxwtGyLDP4KMwYO0I\n4XKU9NMCt4HjD+ZYfAOIBJ8FRVUqM12Splc9Sj8Wp6bVG/5JhDMNpC78CW3qfBG2\nZfV7Nm8jRC5CH6Rcoo4EU8xZJaa4iMt0PPjCwmSDC+0VAGz7M9I/HasLQm9Ca3iV\nah/qag1IrCMfC16gQBXro7RCafBkOQJpI4v6hOIZgB/mHEJjmrsNHAkc312mSvKL\noeaHS7KtM+iDZOrlcPtBF7lqSMwflqghXDKxHFYgyL9VvfX5p/QRD7rxmbB7I3+3\nGQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:46:48,951 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:46:48,951 DEBUG core.py Разбор сообщения от клиента : {'action': 'pubkey_need', 'time': 1647190008.5963457, 'account_name': 'test2'}
2022-03-13 21:46:48,954 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>, {'response': 511, 'bin': '-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA96dVf6GERWiEbawc/ssK\non7bIsbtM5Oh6/U9GG3aFX6oJhx2qNWbcmvDcR5dF+Me0vRCM8/7aKNegQc2B646\n70x6hK5wJIyleVy9XElMnGMRBPyx3/ymFURqyGdlTdcrpUxeNqq16a1VAyDqiOmW\nowg8Tl8FSRz3qn8J0uZDrgiJIF+SbVJw0eHySpopKMhYLI5+pf8ucXJ+LMSHCXwp\ngUsiEFaVFfp4AanSTb8F8ILKvKnL8hoAqyXnQiipA3szk5t3ZX1fvlFRFmU6jFZ5\nstOYxtQDtivga3mhVFp3aCB9dxoZw8I5b3RXwd12wQj1KdqRjX4sb1hiLWr7jkz4\niQIDAQAB\n-----END PUBLIC KEY-----'}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:46:58,148 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:46:58,148 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1647190017.5129118, 'mess_text': 'TMCMeJ7Uc4w1mnEmaNWBfbL4kybKtXjxHsiV+Xdkqmi6Qvh7jrTD4Db4S86rpdWMwfqxHDvGRp/BSGJzLen0TjbJNZig5XzBVFP7A/0MKyar2+rrvnXZbPOB0YbcCZ5c5ORrpraJ0U1GH2yEKQ2pL0VPixGWHnJVkDCp8VGcKsStvddMMBpLqqSynD9ihPl7Yexc6IKmvcWSGffeSPy7ArwS5AVys6hTlCBdi9+4d5GfQCZCh2O/2bR4pxKlC9SmSZIN0UbPQicvm5BHQYX+lfZtFP6/h7/abAOJL+1H+1+kOrEcma/mJml4POK9DYaeec3KzNJQkMYAhennfjdgag=='}
2022-03-13 21:46:58,263 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>, {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1647190017.5129118, 'mess_text': 'TMCMeJ7Uc4w1mnEmaNWBfbL4kybKtXjxHsiV+Xdkqmi6Qvh7jrTD4Db4S86rpdWMwfqxHDvGRp/BSGJzLen0TjbJNZig5XzBVFP7A/0MKyar2+rrvnXZbPOB0YbcCZ5c5ORrpraJ0U1GH2yEKQ2pL0VPixGWHnJVkDCp8VGcKsStvddMMBpLqqSynD9ihPl7Yexc6IKmvcWSGffeSPy7ArwS5AVys6hTlCBdi9+4d5GfQCZCh2O/2bR4pxKlC9SmSZIN0UbPQicvm5BHQYX+lfZtFP6/h7/abAOJL+1H+1+kOrEcma/mJml4POK9DYaeec3KzNJQkMYAhennfjdgag=='}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_message.
2022-03-13 21:46:58,263 INFO core.py Отправлено сообщение пользователю test1 от пользователя test2.
2022-03-13 21:46:58,265 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:47:02,864 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:47:02,864 DEBUG core.py Разбор сообщения от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1647190022.843677, 'mess_text': '7E5SsEUwIJzY4JeSSDKyIiV+K9HLPaNcIQd2N4kiUayhxf+xDFFm7K0lBMCrwLUvgwBV1BXBhou5CyA4H2Rvg5m7ER+KPxakkS274ojno+b5i1jjI6AyYfrbiY/n1zdRwFhb+7QPWH4KF8qUWM6HdIUn+hfrePoSO6SFKxV17YYskr4Jb5WHKVpXok/YTjf8VC+aVDHO2XEWe7rBjgEEuLUUZX5IyhGM4UKenuW3LQKtYJ6fMlkl1wytLauucqe6aCVPbGJGMXZavSp/5L/w888kSes3YFSS2oQtuiP6yRT4OeZ1kfOjRGuXMWxmszoX5ReCHnXq+jCOHNBGCPSStg=='}
2022-03-13 21:47:03,000 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>, {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1647190022.843677, 'mess_text': '7E5SsEUwIJzY4JeSSDKyIiV+K9HLPaNcIQd2N4kiUayhxf+xDFFm7K0lBMCrwLUvgwBV1BXBhou5CyA4H2Rvg5m7ER+KPxakkS274ojno+b5i1jjI6AyYfrbiY/n1zdRwFhb+7QPWH4KF8qUWM6HdIUn+hfrePoSO6SFKxV17YYskr4Jb5WHKVpXok/YTjf8VC+aVDHO2XEWe7rBjgEEuLUUZX5IyhGM4UKenuW3LQKtYJ6fMlkl1wytLauucqe6aCVPbGJGMXZavSp/5L/w888kSes3YFSS2oQtuiP6yRT4OeZ1kfOjRGuXMWxmszoX5ReCHnXq+jCOHNBGCPSStg=='}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_message.
2022-03-13 21:47:03,000 INFO core.py Отправлено сообщение пользователю test2 от пользователя test1.
2022-03-13 21:47:03,001 DEBUG decorator.py Была вызвана функция send_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>, {'response': 200}), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции process_client_message.
2022-03-13 21:47:06,577 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64264)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:47:06,577 DEBUG core.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647190026.3289316, 'account_name': 'test1'}
2022-03-13 21:47:06,577 INFO core.py Клиент ('127.0.0.1', 64264) отключился от сервера.
2022-03-13 21:47:07,205 DEBUG decorator.py Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 7777), raddr=('127.0.0.1', 64265)>,), {}.
Вызов из модуля "C:\Users\Necris45\AppData\Local\Programs\Python\Python39\lib\threading.py",.
Вызов из функции run.
2022-03-13 21:47:07,205 DEBUG core.py Разбор сообщения от клиента : {'action': 'exit', 'time': 1647190027.137148, 'account_name': 'test2'}
2022-03-13 21:47:07,205 INFO core.py Клиент ('127.0.0.1', 64265) отключился от сервера.
2022-03-13 21:47:25,258 DEBUG decorator.py Была вызвана функция main c параметрами (), {}.
Вызов из модуля "E:\GB_Code\Client-server_app\python_chat\server.py",.
Вызов из функции <module>.
